debug/linking_utils/ldd.py

```diff
@@ -1,9 +1,18 @@
 import subprocess
 import os
+import sys
+import re
 
 
 ### helper functions
 
+if sys.version_info > (3, 0):
```

Comment:
Why not just `d.items()`? Is the performance difference worth the special case?

Reply:
Does `.items()` work in both python2 and python3?